easybuild/easyblocks/generic/rpm.py

```diff
@@ -91,58 +128,29 @@ def configure_step(self):
             ver = res.groupdict()['version']
 
             if LooseVersion(ver) >= LooseVersion('4.8.0'):
```

Comment:
yes, because that version has become stricter... installing RPM under a prefix was easier with SL5 (older version of `rpm`)